src/scatter_buildblock/ScatterSimulation.cxx

```diff
@@ -215,14 +219,15 @@ void
 ScatterSimulation::set_defaults()
 {
     this->attenuation_threshold =  0.01f ;
+    // CHANGED THIS TO FALSE
     this->randomly_place_scatter_points = true;
     this->use_cache = true;
     this->zoom_xy = -1.f;
     this->zoom_z = -1.f;
     this->zoom_size_xy = -1;
     this->zoom_size_z = -1;
     this->downsample_scanner_bool = false;
-    this->downsample_scanner_dets = 64;
+    this->downsample_scanner_dets = -1;
```

Comment:
will break things. we should keep the old default I think. Can't this be set via the .par file?